test/api/external/ConfigTest.java

```diff
@@ -0,0 +1,38 @@
+package api.external;
+
```

Comment:
Please remove newlines

Reply:
Its there only to separate package from imports. A general following.